content/applications/essentials/contacts.rst

```diff
@@ -0,0 +1,243 @@
+- :guilabel:`Title`: select :guilabel:`Doctor`, :guilabel:`Madam`, :guilabel:`Miss`,
+  :guilabel:`Mister`, :guilabel:`Professor`, or create a new one directly from this field.
+- :guilabel:`Tags`: enter preconfigured tags by typing them in the field, or clicking the drop-down
+  menu, and selecting one. To create a new one, type the new tag in the field, and click
+  :guilabel:`Create` from the resulting drop-down menu.
+
+Contacts & Addresses tab
+------------------------
+
+At the bottom of the contact form are several tabs. On the :guilabel:`Contacts & Addresses` tab,
+contacts can be added that are associated with a :guilabel:`Company` and related addresses. For
+example, a specific contact person for the company can be listed here.
+
+Multiple addresses can be added on both :guilabel:`Individual` and :guilabel:`Company` contacts. To
+do so, click :guilabel:`Add` in the :guilabel:`Contacts & Addresses` tab. Doing so reveals a
+:guilabel:`Create Contact` pop-up form, in which additional addresses can be configured.
+
+.. image:: contacts/contact-form-add-address.png
+   :align: center
+   :alt: Add a contact/address to the contact form.
+
+On the :guilabel:`Create Contact` pop-up form, start by clicking the default :guilabel:`Other
+Address` field at the top to reveal a drop-down menu of address-related options.
+
+Select any of the following options:
+
+- :guilabel:`Contact`: adds another contact to the existing contact form.
+- :guilabel:`Invoice Address`: adds a specific invoice address to the existing contact form.
+- :guilabel:`Delivery Address`: adds a specific delivery address to the existing contact form.
+- :guilabel:`Other Address`: adds an alternate address to the existing contact form.
+- :guilabel:`Private Address`: adds a private address to the existing contact form.
+
+Once an option is selected, enter the corresponding contact information that should be used for the
+specified address type.
+
+.. image:: contacts/create-contact-window.png
+   :align: center
+   :alt: Create a new contact/address on a contact form.
+
+Add the :guilabel:`Contact Name`, :guilabel:`Address`, :guilabel:`Email`, along with the
+:guilabel:`Phone` and/or :guilabel:`Mobile` numbers below.
+
+Set the :guilabel:`Job Position`, which appears if the :guilabel:`Contact` address type has been
+selected. This is similar to the :guilabel:`Individual` contact.
+
+To add a note, click on the text field next to :guilabel:`Notes`, and write anything that is
+applicable to the customer or contact.
+
+Then, click :guilabel:`Save & Close` to save the address, and close the :guilabel:`Create Contact`
+window. Or, click :guilabel:`Save & New` to save the address, and immediately input another one.
+
+Sales & Purchase tab
+--------------------
+
+Next, is the :guilabel:`Sales & Purchases` tab, which only appears when the *Sales*, *Purchase*,
+**or** *Point of Sale* applications are installed.
+
+Below that, the :guilabel:`Fiscal Position` can be set.
```

Comment:
"Below that" doesnt make a lot of sense to me. It would be helpful to use more specific language, as well the remaining Fiscal Position instructions come after the tabs... lets keep these instructions together :)
```suggestion
The :guilabel:`Fiscal Position` can be set on the :guilabel:`Sales & Purchases` tab. Select a
:guilabel:`Fiscal Position` from the drop-down menu.
```